kingpin/actors/aws/ecs.py

```diff
@@ -361,7 +959,7 @@ def _list_task_definitions(self, status='ALL', family_prefix=None):
         raise gen.Return(task_definitions)
 
     @staticmethod
-    def _load_task_definition(task_definition_file, tokens, default_tokens={}):
+    def _load_task_definition(task_definition_file, tokens, default_tokens=None):
```

Comment:
using `{}` in default params is a potential bug https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
I feel like we used to have issues with setting this to `None` as well in the past.

Reply:
It did make the line too long haha. Was that the issue?